**E1.** El índice de masa corporal (IMC) es una medida de la obesidad y este se cálculo mediante la formula
$$
IMC = \frac{P}{A^2}
$$
donde $P$ es el peso en kilogramos y $A$ es la altura en metros. La clasificación de obesidad es:


<18.5: Bajo peso

18.5-24.9: Normal

25.9-29.9: Sobrepeso

\>29.9: Obesidad

**E1.1** Escriba un programa que calcule el IMC de una persona. Se debe pedir all usuario su peso en kilogramos y su altura en centímetros. Como resultado se debe desplegar un enunciado que diga "Tu IMC es VVV, cuya clasificación es CCC", donde VVV es el valor de IMC calculado, y CCC es la clasificación correspondiente

**Solucion:** El programa calcula el IMC de una persona solicitando su estatura y peso, con ellos calcula el IMC de la persona y le asigna una clasificacion en base a una serie de instrucciones definidas.

In [43]:
import math as ma
p = float(input("Dame tu peso en Kg: "))
e = float(input("Dame tu estatura en cm: "))
IMC = p/((e/100)**2)
if IMC < 18.5 : print(f"Tu IMC es {IMC:2.4} cuya clasificación es: Bajo peso")
elif 18.5<IMC< 24.9 : print(f"Tu IMC es {IMC:2.4} cuya clasificación es: Normal")
elif 25<IMC< 29.9 : print(f"Tu IMC es {IMC:2.4} cuya clasificación es: Sobrepeso")
elif 29.9<IMC: print(f"Tu IMC es {IMC:2.4} cuya clasificación es: Obesidad")

Dame tu peso en Kg: 68
Dame tu estatura en cm: 155
Tu IMC es 28.3 cuya clasificación es: Sobrepeso


**E1.2** Pruebe su programa calculando el IMC y obteniendo la clasificación para las personas con los siguientes datos:    
a) Altura: 188cm, peso: 82kg                
b) Altura: 155cm, peso: 68kg      
Reporte en una celda Markdown sus resultados

Persona **a)** cuya altura es: 188 cm y peso: 82 Kg

IMC y clasifiación reportada: IMC= 23.2, Clasificación: Normal

Persona **b)** cuya altura es: 155 cm y peso: 68 Kg

IMC y clasifiación reportada: IMC= 28.3, Clasificación: Sobrepeso